WebRoot/js/zimbraMail/share/view/dialog/ZmRevokeShareDialog.js

```diff
@@ -74,7 +74,7 @@ function(share) {
 
 	var params = isPubShare ? ZmMsg.shareWithPublic : isGuestShare ? share.grantee.id : isAllShare ? ZmMsg.shareWithAll :
 					(share.grantee.name || ZmMsg.userUnknown);
-	this._confirmMsgEl.innerHTML = this._formatter.format(params);
+	this._confirmMsgEl.innerHTML = this._formatter.format(AjxStringUtil.htmlEncode(params));
```

Comment:
encoding is only needed when we have a placeholder in the string
to remove confusion I have moved encoding to only data part